homeassistant/components/ozw/strings.json

```diff
@@ -4,17 +4,26 @@
       "on_supervisor": {
         "title": "Select connection method",
         "description": "Do you want to use the OpenZWave Supervisor add-on?",
-        "data": {"use_addon": "Use the OpenZWave Supervisor add-on"}
+        "data": { "use_addon": "Use the OpenZWave Supervisor add-on" }
       },
       "install_addon": {
         "title": "The OpenZWave add-on installation has started"
       },
       "start_addon": {
         "title": "Enter the OpenZWave add-on configuration",
-        "data": {"usb_path": "[%key:common::config_flow::data::usb_path%]", "network_key": "Network Key"}
+        "data": {
+          "usb_path": "[%key:common::config_flow::data::usb_path%]",
+          "network_key": "Network Key"
+        }
+      },
+      "hassio_confirm": {
+        "title": "Set up OpenZWave integration with the OpenZWave add-on",
+        "description": "Do you want to configure Home Assistant to connect to the OpenZWave daemon provided by the Supervisor add-on?"
```

Comment:
I don't think that this description adds anything that isn't already in the title.
```suggestion
```